src/openrct2-ui/windows/Changelog.cpp

```diff
@@ -73,7 +73,12 @@ class ChangelogWindow final : public Window
         {
             throw std::runtime_error("Unable to open " + path);
         }
-        return std::string((std::istreambuf_iterator<char>(fs)), std::istreambuf_iterator<char>());
+        auto length = fs.tellg();
+        char* buffer = new char[length];
```

Comment:
Use unique_ptr here, read might throw and it would have a memory leak, same goes for construction of std::string

Reply:
Thanks, will do tomorrow

Reply:
done